tests/autodiff/intellisense-test.slang

```diff
@@ -0,0 +1,39 @@
+RWStructuredBuffer<float> outputBuffer;
```

Comment:
If this is intended to test language server, use `//TEST:LANG_SERVER:` and place it under `tests/language-server`, you can refer to other tests there to see how to test the language server features.